runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -88,6 +92,30 @@ public TokenResponse next(UUID ... streamIds) {
         }
     }
 
+    /**
+     * Retrieve a stream's address space from sequencer server.
+     *
+     * @param streamsAddressesRange range of streams address space to request.
+     * @return address space composed of the trim mark and collection of all addresses belonging to this stream.
+     */
+    public StreamAddressSpace getStreamAddressSpace(StreamAddressRange streamsAddressesRange) {
+        return getMultipleStreamsAddressMap(Arrays.asList(streamsAddressesRange)).get(streamsAddressesRange.getStreamID());
+    }
+
+    /**
+     * Retrieve multiple streams address space.
+     *
+     * @param streamsAddressesRange list of streams ranges to be requested.
+     * @return address space for each stream in request.
+     */
+    public Map<UUID, StreamAddressSpace> getMultipleStreamsAddressMap(List<StreamAddressRange> streamsAddressesRange) {
```

Comment:
Is getStreamsAddressMap a more appreciate name? getMultipleStreamsAddressMap is not consistent with other parts.